README.md

```diff
@@ -11,7 +11,7 @@
 
 ## Live Demo (if available)
```

Comment:
[OPTIONAL] I recommend you remove the `(if available)` portion of the header since you now have a Live Demo.